fishjam_protos/.formatter.exs

```diff
@@ -1,4 +1,5 @@
 # Used by "mix format"
 [
-  inputs: ["{mix,.formatter}.exs", "{config,lib,test}/**/*.{ex,exs}"]
+  inputs: ["{mix,.formatter}.exs", "{config,lib,test}/**/*.{ex,exs}"],
+  locals_without_parens: [field: 1, field: 2, field: 3, field: 4, oneof: 1, oneof: 2]
```

Comment:
Maybe instead we could import this config from `protobuf` package:
`import_deps: [:protobuf]` ?